chapter3.adoc

```diff
@@ -1,71 +1,63 @@
 [[dmsext]]
 == Debug Module Security Extension (non-ISA extension)
 
-This chapter outlines the security enhancements implemented in the Debug Module. The debug operations from external debuggers will be prohibited when the debug capability is not allowed by mdbgen[i] and `sdedbgalw`.
+This chapter outlines the security enhancements defined for the Debug Module as non-ISA extension. The debug operations listed below are modified by the non-ISA extension. All features in this chapter must be implemented in Debug Module to achieve external debug security. If any hart in the system implements the Sdec extension, the Debug Module must also implement the non-ISA extension.  
```

Comment:
- Typo Sdec instead of Sdsec
- The bullet list below is not connected to this paragraph

Reply:
Thanks for spotting it. Will fix typo and add the missing title for the bullet list.